modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/resources.jsp

```diff
@@ -53,7 +53,7 @@ String editorName = (String)request.getAttribute(AlloyEditorConstants.ATTRIBUTE_
 				alloyEditorInstances = 0;
 				alloyEditorDisposeResources = false;
 
-				if (!Object.keys(CKEDITOR.instances)) {
```

Comment:
Introduced in 7ce3ed4772366d6d354b6bd1e0fa3323fa0fd30e.

cc @marcelabc

I made a ticket over here to catch this kind of thing in the linter: https://github.com/liferay/eslint-config-liferay/issues/150

Reply:
Thank you for tracking and fixing this issue :)